### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by TatriX** [photodiode/panorama-view#76]
Hi! Nice addon. Could you please describe the current project status? Are you going to maintain and improve it?

**Comment by JackAdams** [comment]
Yes. This is a question I'm interested in too. This extension is so close to being completely awesome (like the old Tab Groups).

In particular, are some of the nice features from "Totally not Panorama" going to be implemented, like:
- Click and drag to open a tab group
- Drag a tab to some empty space to start a new group

**Comment by Sada-Mio** [comment]
- Right click context menu to send tab to another group. So drag drop one tab at a time isn't needed
- Horizontal auto expanding groups like [this ](https://addons.cdn.mozilla.net/user-media/previews/full/179/179682.png?modified=1485523407). With many groups it becomes tedious to manually resize

**Comment by photodiode** [comment]
Hi
I do plan on improving this add-on. There are many small things I want to implement.
But I haven't had to time to work on it for a while now..
Hopefully I'll get some work done in not too long.

**Comment by Velociraptor85** [comment]
@photodiode how is it going :)? just watned to bump this up since I really would like to use it but cant really because some annoying issues others already documented ... would be nice to see something from you :) and thanks for your work! 👍

**Comment by photodiode** [comment]
I am sorry for the long silence.
Currently working on a rewrite of the entire add on.
This should hopefully fix a lot of the bugs and make further development much easier.
Sorry for doing off github.

**Comment by dabat** [comment]
@photodiode can you tell me why you and @projectdelphai aren't just working together on a single solution?
https://github.com/projectdelphai/panorama-tab-groups
i'm confused why he forked this repo, and now that version seems to be much better maintained.
why not work together?
